corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -151,7 +151,7 @@ <h4>{% trans "[Support only] Assigned Locations" %}</h4>
                   {% endblocktrans %}
                 {% endif %}
               </p>
-              <p>
+              <div style="margin-top: 5px;">
```

Comment:
I _know_ inline styles are not ideal, but in this case I think it would be weirder to create a new styling class just for this one element. Bootstrap 5 will make this easier to deal with in the future.

Reply:
There's a [compatibility file](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/less/b5-compatibility.less) to copy styles from B5 for usage on B3 pages. Would that work here?

Reply:
Yes, that's doable using `mt-2` in this case. Do you think it's worth updating the other inline styles on this page that way too? (there are only 3 total)

Reply:
yes please!

Reply:
Updated the spacing styles: https://github.com/dimagi/commcare-hq/pull/35885/commits/381cb6b9c639a34e32468c38a91f6105693b7e09

There's one inline style I left that just applies a monospace font in this table 
![image](https://github.com/user-attachments/assets/539cabcb-cd8a-4e5e-ac46-64738133ef70)
The equivalent B5 style `font-monospace` uses a variable for the font-family that I wasn't sure how to apply in a less file.

Reply:
Cool, the eventual B5 migration will flag that and it'll be a no-thinking-required change. Thank you for making that future migration a little bit easier.